e2e/server_test.go

```diff
@@ -70,6 +121,71 @@ func TestOptimismClientWithGenericCommitment(t *testing.T) {
 	require.Equal(t, testPreimage, preimage)
 }
 
+func TestProxyClientIntegrationOnly(t *testing.T) {
```

Comment:
Are these tests really testing the Client? I feel like they are testing the server instead?

Reply:
Yep thanks, that works for me :)